runner/run_query.py

```diff
@@ -320,9 +322,10 @@ def ssh_spark(command):
   # Two modes here: Spark SQL Mem and Spark SQL Disk. If using Spark SQL disk use
   # uncached tables. If using Spark SQL Mem, used cached tables.
 
-  query_list = "set spark.sql.codegen=true; set spark.sql.shuffle.partitions = %s;" % opts.reduce_tasks
-  # Comment out for uncompressed output.
-  query_list += "SET hive.exec.compress.output=True;SET io.seqfile.compression.type=BLOCK;"
+  query_list = "set spark.sql.codegen=true;set spark.sql.shuffle.partitions=%s;" % opts.reduce_tasks
+  if (opts.compress):
```

Comment:
Does it make sense to tie this to the input format? I was imagining you'd change this automatically based on the input format.